demo_project/demo/utils.py

```diff
@@ -578,3 +578,47 @@ def create_demo_mongo():
                       ['Punjab',3467,8900,5670,9900]
                       ]
 
+funnel_data = [['Unique users', 'Counts'],
+            ['Website visits', 654],
+            ['Downloads', 4064],
+            ['Requested price list', 1987],
+            ['Invoice sent', 976],
+            ['Finalized', 846]
+        ]
+
+
+treemap_data = [["Continent","Country","Cause","Death Rate"],
```

Comment:
@yashrastogi16 There already is treemap_data in this file which is used by Gcharts. Let's name this treemap_data_highcharts.